source/auth/auth.rst

```diff
@@ -1210,8 +1210,8 @@ Drivers need to support both the shorter and longer SCRAM-SHA-1 and SCRAM-SHA-25
 | S: :javascript:`{conversationId : 1, payload: BinData(0,"cj1meWtvK2QybGJiRmdPTlJ2OXFreGRhd0xIbytWZ2s3cXZVT0tVd3VXTElXZzRsLzlTcmFHTUhFRSxzPXJROVpZM01udEJldVAzRTFURFZDNHc9PSxpPTEwMDAw"), done: false, ok: 1}`
 | C: :javascript:`{saslContinue: 1, conversationId: 1, payload: BinData(0, "Yz1iaXdzLHI9ZnlrbytkMmxiYkZnT05Sdjlxa3hkYXdMSG8rVmdrN3F2VU9LVXd1V0xJV2c0bC85U3JhR01IRUUscD1NQzJUOEJ2Ym1XUmNrRHc4b1dsNUlWZ2h3Q1k9")}`
 | S: :javascript:`{conversationId: 1, payload: BinData(0,"dj1VTVdlSTI1SkQxeU5ZWlJNcFo0Vkh2aFo5ZTA9"), done: false, ok: 1}`
-| *C: :javascript:`{saslContinue: 1, conversationId: 1, payload: BinData(0, "")}`*
-| *S: :javascript:`{conversationId: 1, payload: BinData(0,""), done: true, ok: 1}`*
+| **C**: :javascript:`{saslContinue: 1, conversationId: 1, payload: BinData(0, "")}`
+| **S**: :javascript:`{conversationId: 1, payload: BinData(0,""), done: true, ok: 1}`
```

Comment:
I understand the italicization prevented `:javascript:` from being formatted (see: [existing rendering](https://github.com/mongodb/specifications/blob/497fc5723e1d70029a3fe3866a241afb04b34be8/source/auth/auth.rst#backwards-compatibility)). With this change, I think the "(extra round trip italicized)" in the preceding paragraph should be changed to "(extra round trip emphasized)".

Additionally, the vertical pipe character seems to have no effect on actually indenting these lines, so I'd propose adding a single space before each of these lines, as in:

```suggestion
 | **S**: :javascript:`{conversationId: 1, payload: BinData(0,""), done: true, ok: 1}`
```

Reply:
Thanks for the review, I've just fixed it.